src/components/LatestChanges/index.js

```diff
@@ -0,0 +1,215 @@
+        <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-8">
+          What’s changing and when?
+        </h2>
+        <p className="govuk-!-margin-bottom-5">
+          We are introducing new functionality into Repairs Hub so you will be
+          able to add and view photos on a work order.
+        </p>
+        <p className="govuk-!-margin-bottom-5">
+          This change will be live in Repairs Hub from Tuesday 6 August 2024.
+        </p>
+        <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-8">
+          How will it impact how I do my job?
+        </h2>
+        <p className="govuk-!-margin-bottom-5">
+          We are asking <strong>operatives</strong> to add relevant photos of
+          the job when closing a work order:
+        </p>
+        <ul className="govuk-list govuk-list--bullet">
+          <li>
+            This is for all circumstances - whether the repair is complete, if
+            there is further work to be carried out, or if it is a no access
+            job.
+          </li>
+          <li>
+            Operatives can also add photos after they’ve closed a job using RH
+            desktop.
+          </li>
+          <li>
+            We recommend you take photos of the work as you go (eg. before,
+            during and after) and can then add the photos from your Photo
+            Library.
+          </li>
+        </ul>
+        <p className="govuk-!-margin-bottom-5">
+          We are asking <strong>call centre officers (RCC)</strong> to add
+          photos from residents to all new work orders.
+        </p>
+        <ul className="govuk-list govuk-list--bullet">
+          <li>
+            You can use Hackney Upload / DES to provide a link to residents for
+            them to provide photos, which can then be added to a work order and
+            visible to the operative.
+          </li>
+        </ul>
+        <p className="govuk-!-margin-bottom-5">
+          We are asking <strong>contractors</strong> to add photos to a job when
+          closing a work order.
+        </p>
+        <p className="govuk-!-margin-bottom-5">
+          Anyone else who has relevant photos can add them directly onto the
+          work order at any time eg. supervisors.
+        </p>
+        <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-8">
+          Why is this happening? What problems will this solve?
+        </h2>
+        <ul className="govuk-list govuk-list--bullet">
+          <li>
+            A picture tells a thousand words, and at the moment there’s no way
+            of attaching a photo to a work order, so there’s a lot of
+            understanding we are missing out on.
+          </li>
+          <li>
+            Currently, operatives are often taking photos of their repairs work,
+            and residents are sometimes uploading photos, but these are lost
+            track of with no clear place to store them.
```

Comment:
Currently, repairs work photos taken by residents, and occasionally users are either getting lost, or are hard to keep track of due to there being no clear place to store them.